src/Softeq.NetKit.Chat.Domain.Services/Configuration/SystemMessagesConfiguration.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+namespace Softeq.NetKit.Chat.Domain.Services.Configuration
+{
+    public class SystemMessagesConfiguration
```

Comment:
I would create ctor and accept config params to avoid data corruption. Nw we have open setter

Reply:
Strongly agree. Done.